src/Sylius/Behat/Context/Cli/InstallerContext.php

```diff
@@ -150,7 +150,7 @@ public function iProvideFullAdministratorData()
      *
      * @return resource
      */
-    protected function getInputStream($input)
+    private function getInputStream($input)
```

Comment:
No BC break here, changing method visibility from protected to private happens only for final classes.

Reply:
:)